DasharoPayloadPkg/DasharoPayloadPkg.dsc

```diff
@@ -109,6 +109,7 @@
   DEFINE PERFORMANCE_MEASUREMENT_ENABLE = FALSE
   DEFINE RAM_DISK_ENABLE                = FALSE
   DEFINE APU_CONFIG_ENABLE              = FALSE
+  DEFINE CAPSULE_ENABLE                 = FALSE

```

Comment:
Can this be passed from coreboot? I tried using `BUILD_STR += -D CAPSULE_ENABLE=TRUE`, but it doesn't change the value. I think this may be broken in the same way as `DASHARO_SYSTEM_FEATURES_ENABLE` wasn't properly written to `PcdShowMenu`. It seems to work properly in FDF, but in DSC all `!if $(CAPSULE_ENABLE) == TRUE` are not taken, regardless of what value has been passed by coreboot.

Reply:
I tested it with `BUILD_STR += -D CAPSULE_ENABLE=TRUE` and `BUILD_STR += -D CAPSULE_ENABLE=FALSE` in coreboot.  Are you sure you're triggering a rebuild of EDK?  I usually do `rm -f build/UEFIPAYLOAD.fd build/coreboot.rom` before `make` to force rebuild of EDK.

Reply:
Never mind, it was my fault after all. I was appending to `BUILD_STR` based on `CONFIG_DRIVERS_EFI_UPDATE_CAPSULES`, but I forgot to pass it in `payloads/external/Makefile.mk` :facepalm: 

> I usually do `rm -f build/UEFIPAYLOAD.fd build/coreboot.rom` before make to force rebuild of EDK.

Same, I've lost too much work in payloads due to `make clean/distclean` before learning this trick.